rust/xaynet-ffi/src/api.rs

```diff
@@ -50,10 +50,8 @@ use tokio::{
     time::Duration,
```

Comment:
I think we could move all this to `lib.rs` no? It seems a bit artificial to have only that one module.